.github/skills/theme-inc-folder-formatter/scripts/inc-formatter.cjs

```
@@ -0,0 +1,729 @@
+				const phpTag = '<?php';
+				const phpTagIndex = content.indexOf(phpTag);
+				
+				if (phpTagIndex !== -1) {
+					// Check if there's already a namespace
+					const namespaceMatch = content.match(/^\s*namespace\s+([^;]+);/m);
+					
+					if (namespaceMatch) {
+						// Replace existing namespace
+						content = content.replace(
+							/namespace\s+[^;]+;/,
+							`namespace ${NAMESPACE};`
```

ESLint

[error] 'NAMESPACE' is not defined.

(no-undef)